appserver/tests/paas/multiple-spe-test/spetest/pom.xml

```diff
@@ -109,7 +109,7 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>javax.persistence</groupId>
+            <groupId>jakarta.persistence</groupId>
```

Comment:
I think this one doesn't exist, see:

https://mvnrepository.com/artifact/jakarta.persistence